core/src/main/java/com/example/util/simpletimetracker/core/mapper/CategoryViewDataMapper.kt

```diff
@@ -100,6 +101,18 @@ class CategoryViewDataMapper @Inject constructor(
         )
     }
 
+    private fun mapRecordTagToTagGroupName(
+        tag: RecordTag,
+    ): String {
+        return tag.name.substringBefore("::", "")
```

Comment:
the best would be to move this to an explicit piece of data, something like a meta-tag (tag for tags, aka RecordTagGroup)

Reply:
There is a lot of coupling between the `name` of the tag and the displayed text. I tried to display the truncated name (to display `Tag` instead of `Group::Tag` in the tag selection list), but then editing the tag did not keep the original “full name”.